state/network.go

```diff
@@ -1,29 +1,21 @@
 package state
 
-import "github.com/ChainSafe/gossamer/p2p"
+import "github.com/ChainSafe/gossamer/common"
 
-type networkState struct {
-	p2p *p2p.Service
-}
+type networkState struct{}
 
 func NewNetworkState() *networkState {
-	return &networkState{
-		// TODO: pass p2p service instance to network state
-		p2p: &p2p.Service{},
-	}
+	return &networkState{}
 }
 
-func (ns *networkState) Health() p2p.Health {
-	// TODO: return Health() of p2p service
-	return p2p.Health{}
+func (ns *networkState) Health() common.Health {
+	return common.Health{}
```

Comment:
leave the TODOs?

Reply:
perhaps open issues with the todos and refer it in the TODO would also be a good idea...

Reply:
this is no longer correct, we do not want (and are not able) to import the p2p service here in state

Reply:
I should have addressed this here, I will update